linera-views/src/map_view.rs

```diff
@@ -199,7 +191,7 @@ where
     /// # })
     /// ```
     pub fn remove(&mut self, short_key: Vec<u8>) {
-        if self.delete_storage_first {
+        if self.deletion_set.contains_key(&short_key) {
             // Optimization: No need to mark `short_key` for deletion as we are going to remove all the keys at once.
```

Comment:
This is probably outdated now? Not sure how to formulate it…

```suggestion
            // Optimization: No need to mark `short_key` for deletion as we are going to remove a range of keys containing it.
```

Reply:
Ok.